crawler/crawler/saveToElasticsearch.go

```diff
@@ -122,4 +124,31 @@ func (repo *Repository) generateSlug() string {
 	return fmt.Sprintf("%s-%s", repo.Pa.CodiceIPA, vendorAndName)
 }
 
+// DeleteByQueryFromES delete record from elasticsearch
+// that will match search sting for publiccode.url field
+func (c *Crawler) DeleteByQueryFromES(search string) error {
```

Comment:
Should this function stay in the `saveToElasticsearch.go` file? Seems counter intuitive to me.

Reply:
good catch! The entire file should represent the entrypoint for ES. I will refactor in another PR by the way.